readthedocs/settings/base.py

```diff
@@ -270,13 +296,19 @@ def USE_PROMOS(self):  # noqa
             'options': {'queue': 'web'},
         },
     }
+    MULTIPLE_APP_SERVERS = [CELERY_DEFAULT_QUEUE]
+    MULTIPLE_BUILD_SERVERS = [CELERY_DEFAULT_QUEUE]
 
     # Sentry
     SENTRY_CELERY_IGNORE_EXPECTED = True
 
     # Docker
     DOCKER_ENABLE = False
+    DOCKER_SOCKET = 'unix:///var/run/docker.sock'
+    DOCKER_BUILD_IMAGES = None
+    DOCKER_LIMITS = {}
```

Comment:
We are missing some defaults here

Reply:
As the `getattr` only had an empty dict as the default so, I added this. Should I add this as the default?
https://github.com/rtfd/readthedocs.org/blob/3d36585aca80e7f79b0c5bfeb1e4f6f1e38b8f06/readthedocs/doc_builder/constants.py#L36

Reply:
Yeah